modules/ROOT/pages/deploy-resource-allocation.adoc

```diff
@@ -90,6 +90,19 @@ The resources allocated to your Mule application determine the application's per
 [NOTE]
 Run performance and load testing on your Mule applications to determine the number of resources to allocate.
 
+== Internal Load Balancer Memory Allocation
+
+Internal load balancer memory requirements are impacted by the number of threads, response time latency, and message sizes. The following internal load balancer memory allocations are recommended:
```

Comment:
So, I guess "internal" refers to "memory"? Then, perhaps the heading above could be "Allocating Memory in an Internal Load Balancer". And in this sentence here, "Internal load balancer memory requirements" could be "Requirements for memory in an internal load balancer".
"Impacted" is too strong a word. How about "affected"?
"response time latency" should be "response-time latency".
How are the requirements affected by these things? You say they are, but then don't explain how, even briefly.
"The following internal load balancer memory allocations are recommended" repeats the long phrase "internal load balancer memory" that began the previous sentence. How about changing the sentence to read like this: "MuleSoft recommends these general guidelines for determining the proper allocation of memory:"?